src/content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -54,11 +67,30 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 		},
 
 		/**
-		 * Get file data element wrapper for current file.
-		 * @return {FileDataElementWrapper}
+		 * Find all ranges that contains specified text.
+		 * @param {String} text
+		 * @return {Range[]}
 		 */
-		getFileData : function () {
-			return FileDataElementWrapper.getFromElement(this._domElement);
+		findTextRanges : function (text) {
+			return WindowHelper.findTextRangesInElement(this._dataDomElement, text);
+		},
+
+		/**
+		 * Create hash for left line if current file.
+		 * @param  {Number} number
+		 * @return {String}
+		 */
+		createLeftLineHash : function (number) {
+			return this.hash + "L" + number;
```

Comment:
I think there is no need to use getter, you should access normally private variable this._hash. Also, "L" and "R" should be stored as constants.